.drone.yml

```diff
@@ -493,7 +501,7 @@ pipeline:
 
         _${DRONE_COMMIT_MESSAGE}_
 
-        {{#if build.tag}}`v{{build.tag}}` | {{/if}}Commit <${DRONE_COMMIT_LINK}|{{build.commit}}> | <{{build.link}}|Build #{{build.number}}>
+        {{#if build.tag}}`v{{build.tag}}` | {{/if}}Commit <${DRONE_COMMIT_LINK}|{{build.commit}}> | <{{build.link}}|Build #{{build.number}}> | <https://s3.amazonaws.com/kcoin-e2e/logs/${DRONE_BUILD_NUMBER}/e2e-build-${DRONE_BUILD_NUMBER}.zip|e2e logs>
```

Comment:
You have to change the acl of the upload to `public`

Reply:
The whole bucket is public

Reply:
Oh wait, maybe not.

Reply:
The current log files show `Access Denied`

Reply:
There.